packages/components/pie-webc-testing/src/helpers/defs.ts

```diff
@@ -1,6 +1,21 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
-export type PropObject = {
-    [key: string]: any;
```

Comment:
This is now an optionally-generic type you can pass the component prop type into. This makes sure you can only assign valid values when creating variations inside the visual and accessibility tests.